src/chat/participant.ts

```diff
@@ -148,8 +150,10 @@ export async function handleChatMessage(
   const toolCallsMade = new Set<string>();
 
   // limit number of iterations to prevent infinite loops
+  const maxIterations = vscode.workspace
+    .getConfiguration("confluent")
+    .get("chat.maxIterations", 10);
```

Comment:
Good addition with the user/workspace setting, but let's make sure this setting ID goes into https://github.com/confluentinc/vscode/blob/main/src/preferences/constants.ts, like
```ts
export const CHAT_MAX_TOOL_ITERATIONS = prefix + "chat.maxIterations";
```